connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -399,7 +398,7 @@ public TopicCreationResponse createOrFindTopics(NewTopic... topics) {
             }
         }
         if (topicsByName.isEmpty()) return EMPTY_CREATION;
-        String topicNameList = Utils.join(topicsByName.keySet(), "', '");
+        String topicNameList = String.join("', '", topicsByName.keySet()).replace("[", "").replace("]", "");
```

Comment:
Why we need those `replace`?

Reply:
Sorry just miss that, would fix soon. Thanks!